### PR TITLE
Support Fish Audio tts

### PR DESCRIPTION
**Pull request by MoeMamdouh**
Add FishAudioTTSService for real-time TTS processing via WebSocket

### CODE REVIEW

File: examples/simple-chatbot/server/bot.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
     RTVIBotTranscriptionProcessor,
     RTVIUserTranscriptionProcessor,
 )
-from pipecat.services.elevenlabs import ElevenLabsTTSService
+from pipecat.services.fish import FishAudioTTSService
```
[the remark this answers is not in the document]
Comment (MoeMamdouh) [reply examples/simple-chatbot/server/bot.py]: @markbackman  Done

**Comment by aconchillo** [comment]
@MoeMamdouh Do you mind fixing the formatting issues? Take a look at https://github.com/pipecat-ai/pipecat?tab=readme-ov-file#setting-up-your-editor to setup your editor.

**Comment by markbackman** [comment]
@MoeMamdouh I ran this and hit two issues:
1. There is popping at the start of each phrase
2. The last phrase is not being returned from Fish

The first issue can be resolved but the second seems like a Fish Audio issue. Have you noticed this and/or found a fix.

FYI: I started a parallel branch for this that confirms a bit better to the Pipecat TTS style: https://github.com/pipecat-ai/pipecat/pull/899. We'll probably close this one out and work on the other, but before I do so, I want to see if you've resolved issue #294 

(In the PR I posted, I was able to solve the popping issue by filtering on size. There may be a better way, but for now, it's fine.)

**Comment by markbackman** [comment]
@MoeMamdouh I got a response from the Fish team and resolved the issue. I'm going to close this PR and we'll use https://github.com/pipecat-ai/pipecat/pull/899 instead. Thanks so much for helping to get this started. I hope this service works well for you.

**Comment by MoeMamdouh** [comment]
> @MoeMamdouh I got a response from the Fish team and resolved the issue. I'm going to close this PR and we'll use #899 instead. Thanks so much for helping to get this started. I hope this service works well for you.

thank you @markbackman  You are welcome

**Pull request closed without merging**